src/OpenTelemetry.Abstractions/Trace/IDecision.cs

```diff
@@ -0,0 +1,34 @@
+﻿// <copyright file="IDecision.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Trace
+{
+    using System.Collections.Generic;
+
+    public interface IDecision
+    {
+        /// <summary>
+        /// Gets a value indicating whether Span was sampled or not.
+        /// The value is not suppose to change over time and can be cached.
+        /// </summary>
+        bool IsSampled { get; }
+
+        /// <summary>
+        /// Gets a map of attributes associated with the sampling decision.
+        /// </summary>
+        IReadOnlyDictionary<string, object> Attributes { get; }
```

Comment:
could it be IEnumerable<KeyValuePair<string, object>>

Reply:
hm, the main use would be to copy it into the span attributes. So yes, I think it's reasonable. Thanks!